Modules/Quotas/src/Quotas.Domain/Aggregates/Tiers/Tier.cs

```diff
@@ -71,6 +75,8 @@ public IEnumerable<TierQuotaDefinition> AddQuotaForAllMetricsOnQueuedForDeletion
         var quotaDefinition = new TierQuotaDefinition(metricKey, max, period);
         Quotas.Add(quotaDefinition);
 
+        RaiseDomainEvent(new QuotaCreatedForTierDomainEvent(Id, quotaDefinition.Id));
```

Comment:
I just realized that the naming of the events is inconsistent: `QuotaCreatedForTierDomainEvent` vs. `TierQuotaDefinitionDeletedDomainEvent`. Can you rename `QuotaCreatedForTierDomainEvent` to `TierQuotaDefinitionCreatedDomainEvent`?